test/unit/utils.py

```diff
@@ -14,3 +14,22 @@ def roundtrip(testcase, filename, opts=[]):
   testcase.assertEqual(p.stderr, '')
   with open(path, 'rb') as f:
     testcase.assertEqual(str(p.stdout), str(f.read()))
+
+
+def disassemble(self, filename):
```

Comment:
Surely these should be left as members?   They take `self` as arg0 after all.

Reply:
Yes, but I want to share them across multiple `unittest.TestCase` subclasses. I suppose the proper way to do that would be to make all my test cases inherit from a new class that contains these methods.